tests/test_not_found.py

```diff
@@ -27,15 +29,33 @@ def __assert_404_error(resp, check_body=True):
 @pytest.mark.parametrize('request_path', REQUEST_PATH)
 def test_404_get(nhsd_apim_proxy_url, request_path):
     resp = requests.get(f"{nhsd_apim_proxy_url}{request_path}", headers={
-        "Accept": "*/*"
+        "Accept": "*/*",
+        "Content-Type": "application/vnd.api+json"
     })
     __assert_404_error(resp)
 
 
 @pytest.mark.sandboxtest
-@pytest.mark.parametrize('request_path', REQUEST_PATH)
```

Comment:
Any reason why you're not parameterising this anymore? Could remove the functions below.

Reply:
No I think thats ok, I'll approve.